iChef-WebUi/src/app/data-types/filter-criteria.ts

```diff
@@ -0,0 +1,5 @@
+import { Filter } from './filter';
+
+export interface FilterCriteria {
```

Comment:
I would have the filter and filter criteria in the same ts file. Let's have them both in the filter.ts file